frontend/src/App.js

```diff
@@ -22,28 +22,21 @@ const FolderUpload = () => {
   const handleFileChange = (e) => {
     const files = Array.from(e.target.files);
     // check that the folder being uploaded contains atleast one image
-    let imageFlag = false;
-    const imgExtensions = ['.jpg', '.jpeg', '.tif'];
-    if (files.length > 0) {
-      for (let i=0; i< files.length; i++) {
-        const fileName = files[i].name;
-        const fileNameParts = fileName.split('.');
-        const fileExtension = `.${fileNameParts[fileNameParts.length - 1].toLowerCase()}`;
-        if (imgExtensions.includes(fileExtension)) {
-          imageFlag = true;
-          break;
-        }
-      }
-      if (!imageFlag) {
-        alert('Please upload a folder with atleast one image in it');
-      }
-      
-      if (imageFlag) {
-        setSelectedFolders(prevSelectedFolders => [...prevSelectedFolders, files]);
-      }
+    const imgExtensions = ['.jpg', '.jpeg', '.tif', '.tiff'];
```

Comment:
I guess you're not taking the files that not images - we still need to save them